### ISSUE TITLE
what is format parameter? in `:formatter="format"`

### ISSUE THREAD

**Issue by diamond520** [EducationLink/vue-tel-input#83]
May I ask a beginner's question?
What is the parameter "format" in line `:formatter="format" ` within input tag?
I can't find this parameter in other place.

full input tag:
```
    <input
      ref="input"
      v-model="phone"
      type="tel"
      :placeholder="placeholder"
      :state="state"
      :formatter="format"
      :disabled="disabled"
      :required="required"
      :autocomplete="autocomplete"
      :name="name"
      :class="inputClasses"
      :maxlength="maxLen"
      @blur="onBlur"
      @input="onInput"
    >
```

**Comment by iamstevendao** [comment]
@diamond520, good one! It was a prop of `b-form-input` since we used to use `bootstrap` before. We removed the bootstrap dependency but forgot to remove its legacy:
https://github.com/EducationLink/vue-tel-input/commit/3ada71c023aabfb833f07ce839d589720c52d359#diff-029d3271680c99a3ce1bf50ce7e838f8

I will remove it, thank you!

**Comment by iamstevendao** [comment]
Closed by 1312cf537fae330ea34612fd581e85bedbd4ce07
